### ISSUE TITLE
WRI Indonesia - Layout Builder translations issues

### ISSUE THREAD

**Issue by monicapatel21** [wri/wri_sites#103]
- Try out what works with China
- Consider installing both modules and seeing if there's a choice
- Wil thinks using what we use on China's site will help -- seems like asymmetric might be what we need and we might want to retrofit it to China
- If you want to add two people to two pages, you have to add to both

## Dev tasks
- [x] Install https://www.drupal.org/project/layout_builder_st and see how that works and where it's not solving the problems we need solved.

**Comment by monicapatel21** [comment]
from christian in chat: 

wget -mpck --user-agent="" -e robots=off --wait 1 -E http://www.developmentalert.org

**Comment by mariacha** [comment]
This is ready for a look-over:
https://pr-1-wri-indonesia.pantheonsite.io/en/projects/test-en

I made a video about how it works here:

https://user-images.githubusercontent.com/999525/192906188-fd87cb94-efa9-42a9-806b-b870713d29c9.mov

**Comment by wthomaswri** [comment]
Looks good @mariacha. Not sure if i'm missing a step but it doesn't seem to be pulling in the correct content type when i'm trying it out. Here's an example:https://pr-1-wri-indonesia.pantheonsite.io/en/energy (should just be Articles but think it's grabbing everything)

**Comment by mariacha** [comment]
Also happening now is not working.

**Comment by mariacha** [comment]
This is fixed now:
https://pr-1-wri-indonesia.pantheonsite.io/id/energy

Happening Nows are also working, as should a handful of other fields that pull in a listing of items. 

The problem was something I accidentally broke thinking it was a mistake (removing uuids from the install config - usually when they're in there, it's an accident). I've left a note in these files so I don't do it again.

Back for more staging @wthomaswri

**Comment by mariacha** [comment]
TODO: Figure out the home page https://pr-1-wri-indonesia.pantheonsite.io/en/admin/reports/dblog/event/5076

**Comment by mariacha** [comment]
@wthomaswri Homepage should be ok now: https://pr-1-wri-indonesia.pantheonsite.io/id

I had to clear the cache to fix it was all -- but then I figured I'd fix the Topics view and found some errors in that.

If the home page doesn't load once that's deployed, it may need another cache clear.

**Comment by mariacha** [comment]
TODOs here:
1. Fix the Staff -- import but leave as unpublished if the old value for "Active" was unchecked.
2. Re-push up to pr-1 and re-import staff
3. Send to WRI to review
4. When that looks good, copy over the pr-1 database and code to "migrated" multidev
5. Pull down any config changes from Pr-1
6. Merge PR-1 and deploy to live
7. -- once all is ready, nov 28 estimated -- run migration on live.

**Comment by mariacha** [comment]
@wthomaswri The people should look better now:
https://pr-1-wri-indonesia.pantheonsite.io/id/search?keys=&f%5B0%5D=content_type%3Aperson
https://pr-1-wri-indonesia.pantheonsite.io/en/about/experts-staff

**Comment by monicapatel21** [comment]
Issues to look into:

- [ ] On insights title is hyperlinked above hero -- layout progression
- [ ] Not including people no longer on staff on staff page is being glitchy
- [ ] Spanish language issues: 


Goal: Final deployment (content freeze) for Thursday, December 1

**Comment by mariacha** [comment]
@wthomaswri Did you have an example person who should be unpublished handy from the D7 site?

**Comment by wthomaswri** [comment]
yep @mariacha - Reidinar Juliane

**Comment by mariacha** [comment]
Ok, so now Experts and Staff should look a lot more correct. Here's the page on the new site:
https://pr-1-wri-indonesia.pantheonsite.io/id/about/experts-staff

And here's a version of the staff listing on the old site, where I've removed the groupings so it's easier to compare:
https://d9-migrate-wri-indonesia9.pantheonsite.io/en/about/experts-staff-view-migrate

There's only a 2-person difference, and that feels like something that can be fixed during staging.

I also fixed the [Cidades & Mobilidade Sustentável](https://pr-1-wri-indonesia.pantheonsite.io/en/resources/topic/cidades-mobilidade-sustentavel-9777) problem by just deleting that term.
